src/functionmap.jl

```diff
@@ -17,20 +17,26 @@ function FunctionMap{T}(f::F1, fc::F2, M::Int, N::Int;
 end
 
 # additional constructors
-FunctionMap{T}(f, M::Int; kwargs...) where {T}         = FunctionMap{T}(f, nothing, M, M; kwargs...)
-FunctionMap{T}(f, M::Int, N::Int; kwargs...) where {T} = FunctionMap{T}(f, nothing, M, N; kwargs...)
-FunctionMap{T}(f, fc, M::Int; kwargs...) where {T}     = FunctionMap{T}(f, fc, M, M; kwargs...)
+FunctionMap{T}(f, M::Int; kwargs...) where {T} =
+    FunctionMap{T}(f, nothing, M, M; kwargs...)
+FunctionMap{T}(f, M::Int, N::Int; kwargs...) where {T} =
+    FunctionMap{T}(f, nothing, M, N; kwargs...)
+FunctionMap{T}(f, fc, M::Int; kwargs...) where {T} =
+    FunctionMap{T}(f, fc, M, M; kwargs...)
 
 # properties
 Base.size(A::FunctionMap) = (A.M, A.N)
-Base.parent(A::FunctionMap) = (A.f, A.fc)
+Base.parent(A::FunctionMap) = (A.f, A.fc) # is this meanigful/useful?
```

Comment:
IIRC the reason for `parent` was for manual inspection of wrapped maps, because `show` is now rather sparse. It may be used in the package only rarely, though.

Reply:
perhaps any parent-like method would be most useful if it returned all the arguments needed to recreate the object.  at a minimum that would include size too.  maybe a named tuple would be more useful too.
not sure i would use it much though.

Reply:
From Base:
```
help?> Base.parent
  parent(A)

  Return the underlying "parent array”. ...
```
So I see this making sense for `WrappedMap`, but other than that I am not entirely sure. Where is this used?

Reply:
I think we use `parent` only somewhere in `kronecker.jl`, and then potentially in an inconsistent way. It's not obvious what `parent` should return given the wide range of `LinearMap` subtypes. I'd be fine with actually removing it. I introduced this when I cut down the `show` method to show much less details, and then I used it in one or two places as it was handy. But this is not really crucial.

Reply:
Another decision to be made: what shall we do with `parent`? Just remove? I think it's used internally only in one spot in the kronecker code, and could be worked around with a ternary if-clause, as I had it originally.